### PR TITLE
change code to google coding style

### PR DESCRIPTION
**Pull request by minhthuc2502**

### CODE REVIEW

**Comment by minhthuc2502** [comment]
i think to run doxyfile, you need to configure path of source code

**Comment by GaelGuegan** [comment]
? if the path is relative it should work.

**Comment by GaelGuegan** [comment]
Yes but by putting the Doxyfile in your repository, the user should use it here.

**Comment by minhthuc2502** [comment]
Sorry, I forgot the path relative. I updated Doxyfile. Now it can run anywhere

**Comment by GaelGuegan** [comment]
it would be great that you use the command `git rebase master` to update your branch from master instead of merge.

Then `git push -f` to push because the history is rewritten.

**Comment by minhthuc2502** [comment]
You said that i have to do rebase this time or i will try to use rebase for the next time ?
I think that i merged so i will try to use rebase for the next time if i can

File: src/main.cpp [hunk 1]
```diff
@@ -1,143 +1,143 @@
...
+        case 'h':
+          usage();
+          return EXIT_SUCCESS;
+        case '?':
+          if(optopt == 'c' || optopt == 'm' || optopt == 'p' || optopt == 's')
+            fprintf(stderr, "Option -%c requires an argument. \n", optopt);
+          else if (isprint (optopt))
+            fprintf(stderr, "Unknown option -%c.\n",optopt);
+          else
+            fprintf(stderr, "Unknown option character \\x%x.\n", optopt);
```
Comment (GaelGuegan) [inline src/main.cpp:74]: would it be possible to use the logger function here instead of fprintf ?

File: src/main.cpp [hunk 2]
```diff
@@ -1,143 +1,143 @@
...
+      signal(SIGINT, signalHandler);
+      //signal(SIGTERM, signalHandler);
+
+      TransmitionQueue tQueue;
+      gPtrTQ = &tQueue;
+
+      JoystickPS3 PS3;
+      ArmAL5D AL5D;
+
+      if( PS3.Open() == false ) {
```
Comment (GaelGuegan) [inline src/main.cpp:100]: for all the `if` statement can you follow the google coding style : https://google.github.io/styleguide/cppguide.html#Conditionals

`if (condition) {  // Good - proper space after IF and before {. `

File: includes/armAL5D.hpp [hunk 3]
```diff
@@ -1,19 +1,8 @@
-//$Header$
```
Comment (GaelGuegan) [inline includes/armAL5D.hpp:1]: Can you start all your files (.c & .h) with a comment header of this style : 
```
/**
 * @file doxygen_c.h
 * @author My Self
 * @date 9 Sep 2012
 * @brief File containing example of doxygen usage for quick reference.
 *
 * Here typically goes a more extensive explanation of what the header
 * defines. Doxygens tags are words preceeded by either a backslash @\
 * or by an at symbol @@.
 * @see http://www.stack.nl/~dimitri/doxygen/docblocks.html
 * @see http://www.stack.nl/~dimitri/doxygen/commands.html
 */
```

File: src/main.cpp (hunk 2 above)
Comment (minhthuc2502) [reply src/main.cpp]: i corrected the coding style in if

File: includes/armAL5D.hpp (hunk 3 above)
Comment (minhthuc2502) [reply includes/armAL5D.hpp:1]: i added comment header for all files

**Pull request merged**